website/content/docs/use-cases/hcp-hashistack-quickstart.mdx

```diff
@@ -0,0 +1,501 @@
+
+  tag_specifications {
+    resource_type = "instance"
+
+    tags = {
+      Name = "Nomad-server"
+    }
+  }
+
+  user_data = base64encode(templatefile("user-data-server.sh", {
+    nomad_region              = var.region,
+    nomad_datacenter          = var.cluster_name,
+    consul_ca_file            = base64decode(hcp_consul_cluster.xx_2048_consul.consul_ca_file),
+    consul_gossip_encrypt_key = jsondecode(base64decode(hcp_consul_cluster.xx_2048_consul.consul_config_file)).encrypt,
+    consul_acl_token          = hcp_consul_cluster.xx_2048_consul.consul_root_token_secret_id,
+    consul_private_endpoint_url = hcp_consul_cluster.xx_2048_consul.consul_private_endpoint_url,
+    vault_endpoint            = hcp_vault_cluster.xx_2048_vault.vault_private_endpoint_url,
+    vault_token               = vault_token.nomad_server.client_token
+  }))
+
+}
+```
+
+This is not the only way to define Nomad clusters in Terraform.
+For more practice deploying Nomad to EC2 with Terraform, [follow this tutorial](/nomad/tutorials/cluster-setup/cluster-setup-aws).
+
+## Deploy the Infrastructure!
+
+Once you have finished defining configurations (or cloned them from the [repo](https://github.com/xiaolin-ninja/2048_hcp_hashistack)),
+and set the environment variables for HCP and AWS, you are now ready to deploy your infrastructure!
+
+```shell
+$ terraform init
+Initializing the backend...
+Initializing provider plugins...
+...
+Terraform has been successfully initialized!
+...
+```
+
+Once Terraform has been initialized, you can verify that the resources will
+be created using the `plan` command.
+
+```shell
+$ terraform plan
+An execution plan has been generated and is shown below.
+Resource actions are indicated with the following symbols:
+  + create
+ <= read (data resources)
+Terraform will perform the following actions:
+...
+...
+Plan: 11 to add, 0 to change, 0 to destroy.
+```
+
+Finally, this is the magic moment we promised:
+
+```shell
+$ terraform apply
+...
+Do you want to perform these actions?
+  Terraform will perform the actions described above.
+  Only 'yes' will be accepted to approve.
+  Enter a value:
+```
+
+Remember to confirm the run by entering `yes`!
+
+## Set up HCP Waypoint Context
+
+HCP Waypoint is currently in Beta, and a Terraform module to provision Waypoint is still under development.
+
+Head into your HCP UI and [follow this tutorial](/waypoint/tutorials/runners/hcp-runners-on-your-infra#set-up-the-environment)
+to set up your local environment.
+
+Navigate to Waypoint in the HCP dashboard:
+![HCP UI](img/use-cases/hcp-hashistack-quickstart/hcp.png)
+
+Then click on "Install New Runner", which will generate terminal inputs to set up the Waypoint context and install a Runner on your chosen platform.
+In this case, we choose Nomad with a host volume named "wp_runner".
+![Waypoint Install New Runner](img/use-cases/hcp-hashistack-quickstart/waypoint-runner-install.png)
```

Comment:
I could not see this or the subsequent 7 images in Vercel 🤔 I think that there needs to be a `/` preceding `img`.
<img width="715" alt="Screen Shot 2023-02-08 at 12 01 59 PM" src="https://user-images.githubusercontent.com/83741749/217600108-8d9f46f2-6727-412b-a96d-2c108d24b7ec.png">

Reply:
Oh good catch, I assumed it was a rendering issue like with the images in Github markdown preview.